docs/core/tutorials/debugging-with-visual-studio-code.md

```diff
@@ -107,7 +107,7 @@ The **Debug Console** window lets you interact with the application you're debug
 
 The program displays the string that the user enters. What happens if the user doesn't enter anything? You can test this with a useful debugging feature called a *conditional breakpoint*.
 
-1. Right-click (<kbd>Ctrl</kbd>+click on macOS) on the red dot that represents the breakpoint. In the context menu, select **Edit Breakpoint** to open a dialog that lets you enter a conditional expression.
```

Comment:
Just curious, what is the reason not to use `<kbd>` here? I ask because I use it everywhere and now I'm wondering if I shouldn't.

Reply:
Ctrl-click seems to be a special case mixing keyboard and mouse, so I decided to leave it the way it was in the Mac tutorials as they already existed.  I'm happy to change if you've seen Mac tutorials using `<kbd>Ctrl</kbd>-click`

Reply:
ping @BillWagner on how best to render control-click for macOS in tutorials

Reply:
Hmmm. I don't have a clear recommendation. I think the `<kbd>Ctrl</kbd>-click` is best.

Reply:
A related issue:  given that Macs have keyboard labels all in lowercase (enter, control, option, command, etc., should those keys be represented by, e.g., `<kbd>command</kbd>` or `<kbd>Command</kbd>`